src/components/campaign/Campaign.tsx

```diff
@@ -1,16 +1,31 @@
 import * as React from "react";
 import { FC } from "react";
 import { CampaignMatchPOST200Data } from "../../core/dpl-cms/model";
+import { useStatistics } from "../../core/statistics/useStatistics";
 import { LinkNoStyle } from "../atoms/link-no-style";
 import CampaignBody from "./CampaignBody";
 
 export interface CampaignProps {
   campaignData: CampaignMatchPOST200Data;
 }
+
+// In order to see campaigns in development mode, you will most likely need a
+// browser plugin such as Chrome's "Allow CORS: Access-Control-Allow-Origin"
+// in order to bypass CORS policy for dpl-cms data calls
```

Comment:
I understand what you are getting at here but I do not think this is the right place to document this.

I would suggest adding a markdown file describing how to go about working with DPL CMS if you do not have it running locally in [a separate markdown file as a part of the documentation](https://github.com/danskernesdigitalebibliotek/dpl-react/tree/develop/docs).

If you need more inspiration we also have [a lot of similar docs in the CMS project](https://github.com/danskernesdigitalebibliotek/dpl-cms/tree/develop/documentation).

Reply:
Thanks, I made a separate .md file and a separate PR for it 😉